### ISSUE TITLE
Definitions

### ISSUE THREAD

**Issue by rstento** [lightbend/sbt-paradox-project-info#3]
This looks great, and will be very helpful to our users. Looking at the example posted in the email:
![image](https://user-images.githubusercontent.com/22889339/49387049-7fb0bc00-f6e6-11e8-9b01-8c7cea7a8ed8.png)

It might be good to have an asterisk by Java Versions and Scala Versions that links to a definition of what we mean by "supported" versions.  And, this matrix will be much more complicated for products such as the Lightbend Console and Fast Data Platform, might be good to think about their requirements too.

**Comment by ennru** [comment]
Yes, especially for the readiness levels I expect us to show their definitions on the Lightbend web site. 
This is what our use in Alpakka needed, let's take it from here.

**Comment by ennru** [comment]
We've now updated the [Alpakka Kafka site](https://doc.akka.io/docs/akka-stream-kafka/current/home.html#project-info) which uses this plugin.
What do you think is the best way to explain the rows? 
1. Link on the header to lightbend.com
1. Asterisk with explanations below
1. Small explanation text in the same cell
1. Other
